src/strings.js

```diff
@@ -1,25 +1,25 @@
 const sayHello = string => {
-  // your code here
+  return ('Hello, ' + string + '!');
 };
 
 const uppercase = string => {
-  // your code here
+  return string.toUpperCase();
 };
 
 const lowercase = string => {
-  // your code here
+  return string.toLowerCase();
 };
 
 const countCharacters = string => {
-  // your code here
+  return string.length;
 };
 
 const firstCharacter = string => {
-  // your code here
+  return string.slice(0, 1);
```

Comment:
You could also do:

return string[0]

Because strings are sort like an array of characters.